src/plays/calendar/ModalContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react'
+import { Context } from './Context'
+
+const ModalContainer = () => {
+  const context = useContext(Context)
+  const { modalContent, modalTitle, hideModal } = context
+  
+  if (!modalContent) return null
+
+  return (
+    <div className="vincentBCP-calendar-modal" onClick={hideModal}>
```

Comment:
i would suggest making this modal static (i.e.: do not close the modal on the outside click).